### ISSUE TITLE
output to PDF name not working

### ISSUE THREAD

**Issue by wolftune** [sciurius/chordpro#7]
When I run `chordpro file.cho > name.pdf` it creates a file that won't work, but I see __new__.pdf created which works

**Comment by sciurius** [comment]
Yes, output is not written to stdout. I shall add a warning for this case. Or do you think that is really useful to write PDFs to standard output?

**Comment by wolftune** [comment]
* why not use standard output?
* As is, is there a different way chordpro normally specifies the filename?
* By default, shouldn't the filename match the input name instead of being the same output name for all inputs?

**Comment by sciurius** [comment]
* Technically there is no specific reason not to write to standard output. Writing PDFs to standard output is just a not common thing to do.
* The preferred way is to designate the output file using the "--output" command line option. You can use the filename "-" to select standard output.
* ChordPro is often used to produce multi-song PDFs, so wouldn't using the input file name to derive the name of the PDF be confusing?

**Comment by wolftune** [comment]
Thanks, okay, the --output option makes sense enough (I was just used to the standard out thing from chordii I think). I would think that outputting a pdf from a *single* input file should still use the name passed-through in some ways. If the PDF output uses multiple input files, that clearly would not make sense any more.

I don't understand the "1st or 3rd" question, can you rephrase or clarify?

**Comment by sciurius** [comment]
It's about the default output. Should it be standard output (your 1st bullet point) or derived from the input file (3rd bullet point).

**Comment by wolftune** [comment]
I may be confused (I'm still no expert in any of these things). I think if I were to write `> filenameiwant.pdf` that would be respected, but if I did *nothing* to specify, like just did `chordpro somesong.cho` the output would be somesong.pdf

Again, I'm a novice at these things, so don't give my opinions on correct CLI ideas too much weight.

**Comment by sciurius** [comment]
I understand. Actually, this is not a bad suggestion. I'll sleep on it.
Thanks.

**Comment by sciurius** [comment]
Implemented in the dev branch.
